sdk/storage/azure-storage-queue/src/main/java/com/azure/storage/queue/implementation/QueuesImpl.java

```diff
@@ -284,9 +286,9 @@ public QueuesImpl(AzureQueueStorageImpl client) {
      * @return a Mono which performs the network request upon subscription.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<QueuesSetAccessPolicyResponse> setAccessPolicyWithRestResponseAsync(String queueName, List<QueueSignedIdentifier> queueAcl, Integer timeout, String requestId, Context context) {
+    public Mono<QueuesSetAccessPolicyResponse> setAccessPolicyWithRestResponseAsync(String queueName, Iterable<QueueSignedIdentifier> queueAcl, Integer timeout, String requestId, Context context) {
         final String comp = "acl";
-        SignedIdentifiersWrapper queueAclConverted = new SignedIdentifiersWrapper(queueAcl);
+        SignedIdentifiersWrapper queueAclConverted = new SignedIdentifiersWrapper(queueAcl != null ? StreamSupport.stream(queueAcl.spliterator(), false).collect(Collectors.toList()) : null);
```

Comment:
@alzimmermsft  Please look into this.

Reply:
Why we need changes for QueuesImpl? It is auto-gen

Reply:
Can we do the operation an upper layer?

Reply:
Don't change the codegen files, all changes just get blown away when it is regenerated.

Reply:
Doing this in upper layer.